sitewit/services.py

```diff
@@ -25,10 +25,17 @@ def __init__(self, **kwargs):
 
         super(SitewitService, self).__init__(**kwargs)
 
-    def _get_auth_header(self, account_token=None):
+    def _get_account_auth_header(self, account_token):
+        auth_raw = '%s:%s:%s' % (
+            self._partner_id, self._partner_token, account_token)
+
+        return {'PartnerAuth': base64.b64encode(auth_raw)}
+
+    def _get_partner_auth_header(self, subpartner_id=None):
         auth_raw = '%s:%s' % (self._partner_id, self._partner_token)
-        if account_token is not None:
-            auth_raw += ':%s' % account_token
+        if subpartner_id is not None:
+            auth_raw += ':%s' % subpartner_id
```

Comment:
You could use the same pattern here. Building a list, and then joining with colons.